Tests/CalculatorTests.cs

```diff
@@ -78,5 +78,50 @@ public void Divide_WhenByZero_ReturnsAnError()
             var result = Assert.Throws<ArgumentException>(() => _calculator.Divide(2, 0));
             Assert.That(result!.Message, Is.EqualTo("Cannot divide by zero"));
         }
+
+        [TestCase(1)]
+        [TestCase(2)]
+        [TestCase(34)]
+        [TestCase(10989)]
+        public void Power_WhenRaisedToPowerOne_ReturnsFirstNumber(int a)
+        {
+            var result = _calculator.Power(a, 1);
+
+            Assert.That(result, Is.EqualTo(a));
+        }
+
+        [TestCase(3, 9)]
+        [TestCase(7, 49)]
+        [TestCase(45, 2025)]
+        [TestCase(3198, 10_227_204)]
+        public void Power_WhenSquaring_ReturnsExpected(int a, int expected)
+        {
+            var result = _calculator.Power(a, 2);
+
+            Assert.That(result, Is.EqualTo(expected));
+        }
+
+        [TestCase(3, 27)]
+        [TestCase(7, 343)]
+        [TestCase(9, 729)]
+        [TestCase(14, 2744)]
+        public void Power_WhenCubing_ReturnsExpected(int a, int expected)
+        {
+            var result = _calculator.Power(a, 3);
+
+            Assert.That(result, Is.EqualTo(expected));
+        }
+
+        [TestCase(10, 4, 10000)]
```

Comment:
Would this work for power 0s?
`[TestCase(10, 0, 10)]`

Reply:
You're right, it wouldn't. I've refactored now to cover power 0s and negatives.